### ISSUE TITLE
How to obtain the datasets?

### ISSUE THREAD

**Issue by csc-123** [Kid-Liet/IMSE#4]
How to obtain the datasets?

**Comment by kuailexiaohunzi** [comment]
> How to obtain the datasets?

May I ask if the problem has been resolved? The dataset I downloaded did not distinguish between the test set and the training set, and I am not sure if the download was correct
